docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
 
 .. figure:: ../../images/superplot_1.png
     :width: 500px
     :align: center
 
-- Workflow diagrams in help pages are now ``.svg`` rather than ``.png``
+- **New widget and workbench plugin:** :ref:`WorkbenchWorkspaceCalculator` **, allows to perform binary operations and scaling by a floating number on workspaces;
+  This will require your widget layout to be reset when starting workbench v6.2.0 for the first time. Previously saved layouts accessible from** ``View > User Layouts``
+  **may need to be saved again to include the** :ref:`WorkbenchWorkspaceCalculator` **widget.**
+
+.. image::  ../../images/Workbench/WorkspaceCalculator/WorkspaceCalculator.png
+    :align: right
+
+- Workflow diagrams in help pages are now ``.svg`` rather than ``.png``.
 - Peaks can now be added or removed from a PeaksWorkspace using the :ref:`peaks overlay <sliceviewer_peaks_overlay>` in :ref:`sliceviewer`.
-- The list of eligible workspaces in the `WorkspaceSelector` can now be sorted by name
-- New widget and workbench plugin: `WorkspaceCalculator`, allows to perform binary operations and scaling by a floating number on workspaces;
-  This will require your widget layout to be reset when starting workbench v6.2.0 for the first time. Previously saved layouts accessible from ``View > User Layouts``
-  may need to be saved again to include the workspace calculator widget.
-- Added tooltips to all the widgets in the Slice Viewer. Please contact the developers if any are missing.
-- Script editor tab completion and call tip support for Numpy 1.21
-- The visibility of a component parameter in the Pick tab of the InstrumentViewer is now steered by the 'visible' atrribute of a parameter in IPF
+- When fitting a plot, selecting the peak type will only update the default peak shape in the settings if the ``Set as global default`` checkbox is ticked.
 - Plot legends can be shown or hidden from the plot context menu.
-- ADS signal handlers are now synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
 - The plot config dialog notifies the user when there has been an error applying the config to the plot, and allows them to change the config further.
-- When fitting a plot, selecting the peak type will only update the default peak shape in the settings if the "Set as global default" checkbox is ticked.
-- Added help button to the sliceviewer
-- SliceViewer can toggle between different scales again without any issue.
-- SliceViewer uses a more visible divider between the main data view and the peaks table view.
+- The list of eligible workspaces in the `WorkspaceSelector` can now be sorted by name.
+- Added tooltips to all the widgets in the :ref:`sliceviewer`. Please contact the developers if any are missing.
+- Script editor function call tip support for Numpy 1.21.
```

Comment:
I feel like just saying code completion support covers everything, but maybe not.
```suggestion
- Script editor code completion support for Numpy 1.21.
```